src/Sql/dbo/Stored Procedures/Organization_ReadSelfHostedDetailsById.sql

```diff
@@ -0,0 +1,14 @@
+CREATE PROCEDURE [dbo].[Organization_ReadSelfHostedDetailsById]
+    @Id UNIQUEIDENTIFIER
+AS
+BEGIN
+    SET NOCOUNT ON
+        
+    EXEC [dbo].[Organization_ReadById] @Id
+    EXEC [dbo].[OrganizationUser_ReadOccupySeatCountByOrganizationId] @Id
+    EXEC [dbo].[Collection_ReadCountByOrganizationId] @Id
+    EXEC [dbo].[Group_ReadCountByOrganizationId] @Id
+    EXEC [dbo].[Policy_ReadByOrganizationId] @Id
+    EXEC [dbo].[SsoConfig_ReadByOrganizationId] @Id
+    EXEC [dbo].[OrganizationConnection_ReadByOrganizationIdType] @Id, 2 --Scim connection type
```

Comment:
I had expected a single-row response and potentially a new view type.

I'm happy to be wrong/overruled here, but I believe this multi-response query would be a first and seems fairly odd to my eye.

Reply:
I'm not sure what the best solution is here to get a single-row response:

* combine the results of existing sprocs - but it looks like you can't do this without either (a) turning those sprocs into table-valued functions, or (b) creating a temp table and inserting into it - neither of which seem like a good idea

* just write a new query that joins these tables and returns a result - but this would have lots of duplicate data as we join the Organization, Policy, SSOConfig and OrganizationConnection tables

* some middle ground, where we return a single result where there is a 1-to-1 relationship (Organization + all the Counts), and return the rest of the objects separately (Policy, etc)

There is some precedent for returning multi-set results, e.g. https://github.com/bitwarden/server/blob/552befe4b6173458c898cc5a574b3541c18cd946/src/Infrastructure.Dapper/Repositories/CollectionRepository.cs#L44-L46. Although I admit that's at a much smaller scale.

I am not strongly pushing for my current approach, I'm not just not sure of a better one.

Reply:
we spoke on Slack and you said this was fine given existing usages (like the example above)